_data/pages/es/downloads.yml

```diff
@@ -12,6 +17,9 @@ images:
   tar: "/assets/media/icons/tar.png"
   jdbc: "/assets/media/icons/javajdbc.png"
   perftop: "/assets/media/icons/consol.png"
+  windows: "/assets/media/icons/windows.png"
+  zip: "/assets/media/icons/zip.png"
```

Comment:
It's personal preference, of course, but I think the green icon is super garish. Maybe @stevensideyliu has an opinion.

Reply:
I am good with using some other icon as well. Let me know!

Reply:
@stevensideyliu, can you provide your feedback?

Reply:
Updated the icon for zip to use this (icon)[http://www.softicons.com/system-icons/nano-icon-set-by-freeman/zip-files-2-icon]